crates/cli/src/main.rs

```diff
@@ -14,7 +14,13 @@ use eyre::Result;
 #[allow(unreachable_code)]
 #[allow(clippy::needless_return)]
 async fn main() -> Result<()> {
-    let args = Args::parse();
+    let mut args = Args::parse();
```

Comment:
should move this functionality to `parse::partitions::parse_call_datas()` to keep main() cleaner

Reply:
and that way args does not need to be mut